package.json

```diff
@@ -36,19 +37,22 @@
 	"scripts": {
 		"build": "esbuild src/content-script.ts --bundle --minify --outfile=lib/content-script.js --sourcemap",
 		"compile": "tsc",
+		"firefox:dev": "web-ext run",
+		"firefox:package": "web-ext build",
```

Comment:
If `web-ext build` is creating zip files for us then we can remove `script/zip.js`.